drfx/settings.py

```diff
@@ -20,6 +20,8 @@
 
 CUSTOM_INVOICE_REFERENCE_BASE = 10000
 
+MEMBERSHIP_APPLICATION_NOTIFY_ADDRESS = "example@example.com"
```

Comment:
It is good custom to have example emails ending with .invalid, so example@example.com.invalid
This way mail systems don't even try to deliver to it.

Reply:
example.com is a reserved domain name for this purpose rfc2606